api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -124,18 +137,47 @@ def get_census_version_description(census_version: str) -> CensusVersionDescript
     return description
 
 
-def get_census_version_directory() -> Dict[CensusVersionName, CensusVersionDescription]:
+def get_census_version_directory_of_lts_releases(
```

Comment:
What is the rationale for adding this wrapper?  It seems primarily to substitute for docs/examples of how to use `get_census_version_directory`.  It doesn't save the user any keystrokes or required level of understanding....

My suggestion is to remove it, and just improve the an Examples section of `get_census_version_directory`, highlighting suggested use of the `lts` param, etc.

Reply:
This method is intended to be a "policy-aware" convenience method, built upon the "mechanism"-only `get_census_version_directory()` method. I thought this is what we had discussed in the design meeting, but perhaps I misunderstood. 

However, I do think API users read method names (and use them) before they carefully read docs, so it could be worth keeping from a UX perspective.

I'm not wedded to it, but let's hear from @pablo-gar and @mlin first.

Reply:
When I initially made the "separate policy from mechanism" comment, I did not intend for it to imply additional functions, but rather additional operations.  Either approach satisfies that goal.  The question of a separate function name vs. flags is a style question only, and I think we should do whatever is easiest for users to understand/use.